<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewcvdetails</class>
 <widget class="QDialog" name="viewcvdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CV Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/img/Img/logo.jpg</normaloff>:/new/img/Img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 210, 165);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>SELECT STUDENT ID CARD NO. :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 158, 118);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadtable">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>600</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 158, 118);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>SEARCH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gotohome">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>610</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(142, 173, 200);
border-radius:20%;

font: 75 22pt &quot;Rockwell&quot;;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Go To Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/img/Img/logout.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>CV DETAILS</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>168</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>CERTIFICATIONS : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_certifications">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>430</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>168</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>INTERNSHIPS : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_internships">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>240</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>DURATION : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_duration">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>240</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_univ">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_univ">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>UNIVERSITY %:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>168</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>SPECIALIZATION : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_specialization">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>470</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>12TH %: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12thmarks">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>310</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>10TH %</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10thmarks">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
